packages/thirdweb/src/contract/actions/compiler-metadata.ts

```diff
@@ -52,3 +53,31 @@ export function formatCompilerMetadata(metadata: any): CompilerMetadata {
     isPartialAbi: metadata.isPartialAbi,
   };
 }
+
+/**
+ * Formats the compiler metadata into a standardized format.
+ * @param metadata - The compiler metadata to be formatted.
+ * @returns The formatted metadata.
+ * @internal
+ */
+// biome-ignore lint/suspicious/noExplicitAny: TODO: fix later
+export function formatZksolcMetadata(metadata: any): CompilerMetadata {
```

Comment:
do we need a separate function for this? looks exactly the same as the one above. if we dont get the devdoc, then it'll just be undefined thats fine no?